src/pages/folder/container/FolderTagList/FolderTagList.tsx

```diff
@@ -0,0 +1,62 @@
+import styles from "./FolderTagList.module.scss";
+import { FolderInterface } from "@/types";
+
+interface FolderTagListProps {
+  folders: FolderInterface[];
+  current: FolderInterface;
+  onClick: (folder: FolderInterface) => void;
+}
+
+interface FolderTagProps {
+  folder: FolderInterface;
+  onClick: (folder: FolderInterface) => void;
+  clicked?: boolean;
+}
+
+function FolderTag({ folder, onClick, clicked = false }: FolderTagProps) {
+  const handleClick = () => {
+    if (folder) {
+      onClick(folder);
+    }
+  };
+
+  return (
+    <button
+      className={
+        clicked
+          ? `${styles["folder-name-button"]} ${styles["clicked"]}`
+          : `${styles["folder-name-button"]}`
+      }
+      type="submit"
+      onClick={handleClick}
+    >
+      {folder.name}
+    </button>
+  );
+}
+function FolderTagList({ folders, current, onClick }: FolderTagListProps) {
+  return (
+    <div className={styles["folder-tag-list"]}>
+      <div className={styles["folder-tags-container"]}>
+        {folders?.map((folder) => {
+          if (current.id === folder.id) {
+            return (
+              <FolderTag
+                folder={folder}
+                key={folder.id}
+                onClick={onClick}
+                clicked={true}
+              />
+            );
+          } else {
+            return (
+              <FolderTag folder={folder} key={folder.id} onClick={onClick} />
+            );
+          }
```

Comment:
```suggestion
          return (
            <FolderTag folder={folder} key={folder.id} onClick={onClick} clicked={current.id === folder.id}/>
          );
```

clicked 에 바로 boolean 값을 넘겨줘도 괜찮을 것 같습니다.